crates/re_build_tools/src/lib.rs

```diff
@@ -21,6 +23,24 @@ pub use self::rebuild_detector::{
     rerun_if_changed_glob, rerun_if_changed_or_doesnt_exist, write_file_if_necessary,
 };
 
+// ------------------
+
+/// Should we export the build datetime for developers in the workspace?
+///
+/// It will be visible in analytics, in the viewer's about-menu, and with `rerun --version`.
+///
+/// To do so accurately may incur unnecessary recompiles, so only turn this on if you really need it.
+const EXPORT_BUILD_TIME_FOR_DEVELOPERS: bool = false;
+
+/// Should we export the current git hash/branch for developers in the workspace?
+///
+/// It will be visible in analytics, in the viewer's about-menu, and with `rerun --version`.
+///
+/// To do so accurately may incur unnecessary recompiles, so only turn this on if you really need it.
+const EXPORT_GIT_FOR_DEVELOPERS: bool = false;
```

Comment:
I decided not to make these into environment variables right now. We can very easily do so now if we change our minds.